franklaercio/contact-verify
# Add tracking to contact fraud save functionality

# Description
This pull request implements functionality to track the contacts (phone numbers or emails) searched by users. Each number or email searched will be saved in the database, enabling tracking of how many people have received calls or messages from that contact.

# Key Changes:
- Database Changes: A new database table has been introduced to store the contact details and the count of users who have interacted with that contact.
- Frontend Validation: The frontend now validates that only valid phone numbers and emails can be submitted for tracking. Invalid inputs will be flagged before submission.

# How to Test:
- Test with a valid phone number or email to ensure it gets saved in the database correctly.
- Test with an invalid phone number or email to verify that the frontend validation prevents it from being submitted.
- Check the database to ensure the number of interactions is updated each time a contact is used.

# Related Issues:
https://github.com/franklaercio/contact-verify/issues/2
